tests/testthat/test-conjunct_test_linter.R

```diff
@@ -1,107 +1,72 @@
+linter <- conjunct_test_linter()
```

Comment:
this is bad IMO. this object will persist to other test files in the suite if it's not captured inside a `test_that()`. that may be OK if it's a rare variable name / not referenced elsewhere, but `linter` is used in almost all test files. Thus this is prone to errors, and really-hard-to-debug issues as each test file is no longer hermetic.